packages/utilities/psammead-assets/src/svgs/mediaIcons.jsx

```diff
@@ -59,6 +67,23 @@ const mediaIcons = {
       <circle cx="20" cy="14.02" r="5.5" />
     </PhotoMediaIcon>
   ),
+  guidance: (
+    <GuidanceIcon
+      viewBox="0 0 24 24"
+      focusable="false"
+      height="24px"
+      width="24px"
+    >
+      <path
+        fill={C_WHITE}
```

Comment:
`fill=""` isn't needed here, because it inherits the colour from the parent style.